src/components/Navigation.vue

```diff
@@ -62,13 +62,18 @@
 
 <script lang="ts">
 import { defineComponent } from 'vue';
+import {SemesterInfo} from "../helpers/semester";
+
+const currentSemester = SemesterInfo.now();
+const defaultStartSemester = currentSemester.springSemester ? currentSemester.removeSemesters(1) : currentSemester;
```

Comment:
Moving the logic to find the most recent "Herbst Semester" to `SemesterInfo`, maybe behind a function called `getMostRecentFallSemester`, and calling it here would be an improvement.
Would make it easier to understand, what this code does and also keep all semester-related logic inside `SemesterInfo`.

Reply:
I don't have time for this. Feel free to do so.

Reply:
I changed it now, found the motivation somewhere.